graphs/possible_bipartition.py

```diff
@@ -5,8 +5,27 @@ def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n^2)
+        Space Complexity: O(n)
```

Comment:
👍 But I think the time complexity is O(NE) where N is the number of nodes and E the number of edges.  I suppose in a fully connected graph it would approach O(n^2)